community/modules/scripts/ramble-setup/scripts/install_ramble_deps.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Install dependencies for ramble installation
+  become: yes
+  hosts: localhost
+  tasks:
+  - name: Install dependencies through system package manager
+    ansible.builtin.package:
+      name:
+      - python3-pip
+      - git
+  - name: Gather the package facts
+    ansible.builtin.package_facts:
+      manager: auto
+  - name: Install protobuf for old releases of Python
+    when: ansible_facts.packages["python3"][0].version is version("3.7", "<") and ansible_facts.packages["python3"][0].version is version("3.5", ">=")
```

Comment:
What happens if the version is pre 3.5? Maybe we should fail.

Reply:
This is copied from the spack module. But, adding a failure with `ansible.builtin.fail` that fails if python <= 3.5 is found causes it to fail (even if 3.6.8 is on the VM). I'm guessing it tests the system python instead of the installed python.

I'm omitting this for now unless you have a suggestion to make this work better.

Reply:
We can revisit when we redo spack.